.travis.yml

```diff
@@ -1,5 +1,4 @@
 ---
-sudo: false
 before_install:
   - export HARNESS_OPTIONS=j10:c HARNESS_TIMER=1
   - rm .travis.yml
```

Comment:
This file was changed recently in the master branch. Looks like your master branch hasn't been updated with those changes. Probably the easiest option is to manually add the `sudo: false` line to your local file and commit. Another option is to perform a rebase.